ecl/hqlcpp/hqlecl.cpp

```diff
@@ -603,9 +603,19 @@ bool HqlDllGenerator::generateCode(HqlQueryContext & query)
 
 void HqlDllGenerator::addWorkUnitAsResource()
 {
-    StringBuffer wuXML;
-    exportWorkUnitToXML(wu, wuXML, false, false, false);
-    code->addCompressResource("WORKUNIT", wuXML.length(), wuXML.str(), NULL, 1000);
+    bool defaultBinaryWorkunit = true; // MORE: CHANGE before release?
```

Comment:
Is an option to write both (for maximum back-compatibility) worth considering?

Reply:
@ghalliday - were you going to remove the comment?

Reply:
@jakesmith needs discussion, but I think the plan is to back port this to 7.12.x so that old systems can read the new format, and then possibly enable it on 8.6.x